MAINTAINERS.md

```diff
@@ -8,25 +8,26 @@
 
 We would like to have similar behavior regardless of whether a chain launched from a release upgrades a predecessor or starts clean. Since we are currently not in a position where we can upgrade all vats, our release must not include behavioral changes to code running in any vat that it is not explicitly upgrading. The easiest way to accomplish that is by avoiding any changes to code included in vats, however this is not always feasible.
 
-We maintain `release-*` branches which represent a fork of the `master` dev branch at a given point:
-- `release-pismo` is the previous/archived release branch used before the "bulldozer" upgrade which threw away all JavaScript state.
-- `release-mainnet1B` is our current release branch. Despite its name, it has lived beyond the "mainnet-1" phase of the agoric-3 chain.
+We previously maintained `release-*` branches which represented a fork of the `master` dev branch at a given point:
+- `release-pismo` is the original/archived release branch used before the "bulldozer" upgrade which threw away all JavaScript state.
+- `release-mainnet1B` was our previous release branch up until upgrade-15. Despite its name, it lived beyond the "mainnet-1" phase of the agoric-3 chain.
 
-Release branches should only ever be updated when publishing a new release.
+We don't expect these previous release branches to ever be updated again.
+
+Starting with `upgrade-16`, we try to release directly from `master` by creating a `dev-release-*` branch. These branches allow us to work on the release concurrently with other engineering efforts, and are not currently merged back once the release is complete.
```

Comment:
Establishing that both `release-…` and `dev-release-…` branches fall under the "release branch" term:
```suggestion
Starting with `agoric-upgrade-16`, we try to release directly from `master` by creating a `dev-release-*` release branch. These branches allow us to work on the release concurrently with other engineering efforts, and are not currently merged back once the release is complete.
```